blocks/social-share/social-share.css

```
@@ -1,36 +1,108 @@
-/* share event */
-.share-event {
+/* News and Publication Variant */
+.publication .share-event,
+.news .share-event {
     float: left;
     padding: 15px 0 0;
   }
-  
-.share-event p {
+
+.publication .share-event p,
+.news .share-event p {
   margin: 0 0 15px;
   text-transform: uppercase;
   font-family: var(--ff-proxima-light);
   font-weight: 600;
   font-size: 14px;
 }
 
-.social-links ul {
+.publication .social-links ul,
+.news .social-links ul {
   list-style: none;
   margin: 0;
   padding: 0;
   display: block;
 }
 
-.social-links li {
+.publication .social-links li,
+.news .social-links li {
   float: left;
   padding-right: 25px;
 }
 
-.social-links li a i {
+.publication .social-links li a i,
+.news .social-links li a i {
   display: inline-block;
   font-size: 21px;
 }
 
-.social-links.blue-ico ul li a i, .social-links ul li i.addthis-share-button {
+.publication .social-links ul li i,
+.news .social-links ul li i {
   color: var(--button-bg-color);
   line-height: 28px;
   cursor: pointer;
+}
+
+/* Blog Variant */
+.blog .share-event p {
+  display: none;
+}
+
+.blog .share-event ul.button-container {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  position: fixed;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  z-index: 1;
+  transition: all .3s ease-in-out;
+}
```

```suggestion
.blog .share-event ul.button-container {
  list-style: none;
  margin: 0;
  padding: 0;
  display: flex;
  flex-direction: column;
  gap: 4px;
  position: fixed;
  left: 0;
  top: 50%;
  transform: translateY(-60%);
  z-index: 1;
  transition: all .3s ease-in-out;
  visibility: visible;
  opacity: 1;
}

.blog .social-share-hidden .share-event ul.button-container {
  visibility: hidden;
  opacity: 0;
  transform: translateY(-50%);
}
```
@andreituicu the on/off effects are not exactly as the original. this change should allow fade in and out and a slight move up on appear and down on disappear